bridge/bridge.go

```diff
@@ -92,6 +93,14 @@ func (r *Result) SetCompleted() {
 	r.Status = "completed"
 }
 
+// SetJobRunID sets the request's ID to the result's Job Run ID.
+// If "jobRunId" is supplied in the request, use that for the response.
+func (r *Result) SetJobRunID() {
+	if r.JobRunID == "" {
```

Comment:
Prefer it checking the length rather than literal quotes. `len(r.JobRunID) == 0`.